docs/0.4.x/en-US/reference/compilation-times.md

```diff
@@ -4,7 +4,7 @@ As you may have noticed while using Perseus, compiling even very small changes t
 
 The first step in addressing this is to figure out whether your app needs exporting or not. If not, then you should use `#[perseus::main_export]` rather than `#[perseus::main(...)]` in your `lib.rs`, and then you don't need to bring in a server integration, which should speed things up slightly. Notably, if you choose not to do this, there's no actual compile-time difference between `perseus export` and `perseus serve` (there used to be, back in the days of `.perseus/`, but those horrors are gone now). As a general rule, apps that *can* go to static files *should* go to static files, because they'll be faster. (It's also more easily possible to push static files to edge networks in deployment, which usually means faster load times for your users, depending on your hosting provider.)
```

Comment:
Shouldn't that be `main.rs` rather than `lib.rs`?

Reply:
Happy to change that here; it will need it at [PR 189](https://github.com/framesurge/perseus/pull/189) also.

Think it's worth modifying these both to add? Or save for another day? Happy to do either...